pkg/k8s/init.go

```diff
@@ -189,7 +189,7 @@ func Init(conf k8sconfig.Configuration) error {
 		return res.Error()
 	}
 
-	if option.Config.K8sHeartbeatTimeout != 0 {
+	if !option.Config.DryMode && option.Config.K8sHeartbeatTimeout != 0 {
```

Comment:
Please add a comment with the "why?" If you are adding this `option.Config.DryMode` does it mean that it is not possible to unit test the heartbeat logic?